test/core/debug/stats_test.cc

```diff
@@ -151,5 +151,9 @@ int main(int argc, char** argv) {
   int ret = RUN_ALL_TESTS();
   grpc_shutdown();
   return ret;
+#else
+  // Avoid unused parameter warning for conditional parameters.
+  (void)argc;
```

Comment:
For my own knowledge, does this cast an integer to void? Is it safe?

Reply:
Yeah, this basically evaluates something (anything) and ignores the result. https://en.cppreference.com/w/c/language/cast : "If type-name is void, then expression is evaluated for its side-effects and its returned value is discarded, same as when expression is used on its own, as an expression statement." Since the result would then be a dead-value, a simple expression like this would get optimized out.